consul-connect/step2.md

```diff
@@ -1,16 +1,14 @@
-# Launch Demo Dashboard
-
-Next, start a second terminal. You'll start the front-end `dashboard-service` on port `9002`. It will not yet connect to the `counting` service...we will do that in another step.
+Next, start a second terminal. You'll start the frontend service, `dashboard-service`, on port `9002`. It will not yet connect to the `counting` service...you will do that in another step.
```

Comment:
```suggestion
Next, start a second terminal. You'll start the frontend service, `dashboard-service`, on port `9002`. It will not yet connect to the `counting` service, you will do that in another step.
```